src/libsync/vfs/cfapi/cfapiwrapper.cpp

```diff
@@ -906,6 +906,65 @@ OCC::Result<void, QString> OCC::CfApiWrapper::createPlaceholderInfo(const QStrin
     return {};
 }
 
+OCC::Result<void, QString> OCC::CfApiWrapper::createPlaceholdersInfo(const QString &localBasePath, const QList<PlaceholdersInfo> &itemsInfo)
+{
+    const auto stdWStringBasePath = localBasePath.toStdWString();
+    auto cloudEntry = std::make_unique<CF_PLACEHOLDER_CREATE_INFO[]>(itemsInfo.size());
+
+    for(auto itemIndice = 0; itemIndice < itemsInfo.size(); ++itemIndice) {
+        const auto &placeholderInfo = itemsInfo[itemIndice];
+
+        if (placeholderInfo.modtime <= 0) {
+            return {QString{"Could not update metadata due to invalid modification time for %1: %2"}.arg(placeholderInfo.relativePath).arg(placeholderInfo.modtime)};
```

Comment:
can you combine these args here?  (might need to wrap the modtime in a `QString::number`)

```suggestion
            return {QString{"Could not update metadata due to invalid modification time for %1: %2"}.arg(placeholderInfo.relativePath, placeholderInfo.modtime)};
```